common/src/main/java/io/homeassistant/companion/android/common/sensors/DisplaySensorManager.kt

```diff
@@ -26,23 +37,63 @@ class DisplaySensorManager : SensorManager {
             "mdi:cellphone-off",
             docsLink = "https://companion.home-assistant.io/docs/core/sensors#screen-off-timeout-sensor"
         )
+
+        val screenOrientation = SensorManager.BasicSensor(
+            "screen_orientation",
+            "sensor",
+            commonR.string.sensor_name_screen_orientation,
+            commonR.string.sensor_description_screen_orientation,
+            "mdi:screen-rotation",
+            docsLink = "https://companion.home-assistant.io/docs/core/sensors#screen-orientation-sensor",
+            updateType = SensorManager.BasicSensor.UpdateType.INTENT
+        )
+
+        val screenRotation = SensorManager.BasicSensor(
+            "screen_rotation",
+            "sensor",
+            commonR.string.sensor_name_screen_rotation,
+            commonR.string.sensor_description_screen_rotation,
+            "mdi:screen-rotation",
+            docsLink = "https://companion.home-assistant.io/docs/core/sensors#screen-rotation-sensor",
+            updateType = SensorManager.BasicSensor.UpdateType.INTENT

```

Comment:
This feels misleading to me, as it only updates when the rotation changes so much that the orientation changes. Slightly rotating the device like a compass or level doesn't result in sensor updates for every degree (which would be a bad idea, as it would be updating nonstop).